### PR TITLE
Check repository for .yml files using YAMLLintBear

### PR DESCRIPTION
**Pull request by yagyanshbhatia**
Add YAMLLintBear in .coafile, resolve issues raised
closes https://github.com/coala/coala-vs-code/issues/14

### CODE REVIEW

**Comment by gaocegege** [comment]
Thanks for your contribution :-) Is the PR ready to review?

**Comment by yagyanshbhatia** [comment]
yes, @gaocegege

File: .coafile [hunk 1]
```diff
@@ -39,3 +39,8 @@ language = python3
 # 0.8.x (current stable release) needs them.
 ci_keywords, keywords = \#TODO, \# TODO, \#FIXME, \# FIXME
 cs_keywords =
+
+
+[yml]
+bears = YAMLLintBear
+files = *.yml, .ci/*.yml, tests/**/*.yml
```
Comment (gaocegege) [inline .coafile]: There are two warnings:

```
[WARNING][12:59:53] No files matching '/home/ist/code/coala-vs-code/.ci/*.yml' were found. If this rule is not required, you can remove it from section [yml] in your .coafile to deactivate this warning.
[WARNING][12:59:53] No files matching '/home/ist/code/coala-vs-code/tests/**/*.yml' were found. If this rule is not required, you can remove it from section [yml] in your .coafile to deactivate this warning.
```

Could we remove `.ci/*.yml, tests/**/*.yml`?

Comment (yagyanshbhatia) [reply .coafile]: yes, right now there are no yml files there.But there may be in future? should i remove it?

Comment (gaocegege) [reply .coafile]: I think so, we could add them if we need them in the future :-)

Comment (yagyanshbhatia) [reply .coafile]: @gaocegege okay done

**Comment by gaocegege** [comment]
LGTM, I am not sure if @Makman2 reviews the PR, I will merge the PR if he does not :-)

Thanks for your contribution

**Comment by Makman2** [comment]
Yes please squash the commits, they belong together.

And indeed, please reread the newcomer guide and especially the commit guidelines.

**Comment by yagyanshbhatia** [comment]
Yes, I understand. I am currently out, I'll do them asap. Thanks for your reviews!

**Comment by Makman2** [comment]
Why closing @yagyanshbhatia ? ;)

**Pull request closed without merging**